src/Nokia_LCD.cpp

```diff
@@ -248,7 +258,27 @@ void Nokia_LCD::sendCommand(const unsigned char command) {
 
 bool Nokia_LCD::sendData(const unsigned char data) { return send(data, true); }
 
-bool Nokia_LCD::send(const unsigned char lcd_byte, const bool is_data) {
+
+bool Nokia_LCD::updateCursorPosition(const unsigned int x_start_position, const unsigned int x_end_position) {
+    bool out_of_bounds = false;
+
+    mX_cursor = (mX_cursor + 1) % (x_end_position + x_start_position);  // Column
```

Comment:
Can you elaborate a bit on `(x_end_position + x_start_position)`? What does that represent?

Reply:
Sure, I will add a comment